### PR TITLE
Value of try before you buy

### PR DESCRIPTION
**Pull request by mooreds**
Added expert advice and excerpt blog post.

### CODE REVIEW

File: site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: blog-post
+title: How to get the most out of a trial period
```
Comment (dpolstra) [inline site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md:3]: How to get the most out of a free auth trial

File: site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+description: When you are trialing software, how can you make most effective use of this time?
```
Comment (dpolstra) [inline site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md:4]: When you are trialing auth software, how can you make most effective use of this time?

File: site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md (hunk 1 above)
Comment (dpolstra) [inline site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md:3]: Or "How to get the most out of your auth POC"

File: site/learn/expert-advice/identity-basics/try-before-you-buy.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Sometimes, despite a salesperson overcoming all your concerns or a landing page perfectly crafted to speak your language in every bullet point, there's a lingering fear that the product you're looking at just might not be what you're looking for. Everyone has experienced buyer's remorse at some point in their life, and most people I know don't want to go through it again.
+
+So what's the only way to know if you'll truly love a product? Trying it risk-free at no monetary cost to you.
+
+This holds especially true for larger or more long-term purchases. When you're evaluating authentication providers, one of the main building blocks of any software product, you want to make sure you won't regret your choice a few months or years later.
+
+Choosing incorrectly could mean a major rewrite of your application logic. Getting months down the road only to realize that a particular provider can't support all your use cases can really kill developer momentum.
+
+## Advantages of Trying Before You Buy
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/try-before-you-buy.md:20]: Advantages of Trying an Auth Provider Before You Buy

File: site/learn/expert-advice/identity-basics/try-before-you-buy.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+This would be a huge problem leading to a major rewrite or possibly having to drop a key feature of your app. However, if you had built a prototype using a free trial and discovered this limitation ahead of time, you could have simply moved on to choosing a different provider.
+
+### Better Understanding of Costs
+
+Many API-based products such as authentication providers are usage-based, which means as your usage increases, the pricing changes. While looking at the pricing table may have you convinced that you'll be on a free or low-cost plan, actually building out a prototype might reveal otherwise.
+
+There are all sorts of potential hidden costs or costs that you may not have considered from just thinking about your application at a high level. Knowing what this product costs on average for other companies of your scale is helpful.
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/try-before-you-buy.md:40]: There are all sorts of potential hidden costs ~or costs~ that you may not have considered

File: site/learn/expert-advice/identity-basics/try-before-you-buy.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+For example, if you're working with an authentication provider that charges you on a per-authentication basis, you might take a look at your users and how active they are, and estimate how many people will be authenticating every month. With this math, you might conclude that you're well under the threshold for the authentication provider's free plan.
+
+But when you build a prototype, you might notice that every incorrect password attempt counts as authentication, demonstrating that your previous estimates might be severely different from what you should expect for actual usage. You might learn that multiple connections to enterprise identity stores, such as SAML IdPs, cost an arm and a leg.
+
+Catching surprises like this during a free trial can help you know how to factor your costs correctly, as opposed to finding out when your integration is already in production. It allows your development team a chance to think up different ways to architect your solution to fit within these constraints, or even choose a different provider if a workaround or change in architecture isn't going to be possible.
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/try-before-you-buy.md:46]: Catching surprises like this during a free trial can help you ~know~ **_understand_** how

File: site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md:3]: Will change to "How to get the most out of a free auth provider trial"

**Pull request merged**